plugins/rikki/loungeviews/components/playoffoverview/default.htm

```diff
@@ -117,29 +117,25 @@ <h6>Team Description:</h6>
                                 <div class="record">
                                     <div class="opponent opponent-1">
                                         {%if match['model'].teams[0] %}
-                                        {%if match['model'].teams[0].logo is not null%}<div class="logo"><img src="{{match['model'].teams[0].logo.path | resize(64,64)}}" style="position: relative; padding-top:0px;"title="{{match['model'].teams[0].title}}"></div>{%endif%}
-                                        <div class="name">{{match['model'].teams[0].title}}</div>
-                                        {%else%}
-                                        <div class="name disabled">TBD</div>
-                                        {%endif%}
+                                        <div class="logo spoiler">{%if match['model'].teams[0].logo is not null%}<img src="{{match['model'].teams[0].logo.path | resize(64,64)}}" style="position: relative; padding-top:0px;"title="{{match['model'].teams[0].title}}">{%endif%}</div>
```

Comment:
The if statement needs to be inside the div not outside, otherwise any team with no logo does not line up properly and also is revealed in the later playoff rounds due to the discrepancy